code/game/objects/items/twohanded.dm

```diff
@@ -850,3 +850,71 @@
 		C.change_view(CONFIG_GET(string/default_view))
 		user.client.pixel_x = 0
 		user.client.pixel_y = 0
+
+/obj/item/twohanded/broom
+	name = "broom"
+	desc = "This is my BROOMSTICK! It can be used manually or braced with two hands to sweep items as you move."
+	icon = 'icons/obj/janitor.dmi'
+	icon_state = "broom0"
+	lefthand_file = 'icons/mob/inhands/equipment/custodial_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/equipment/custodial_righthand.dmi'
+	force = 8
+	throwforce = 12
+	throw_speed = 3
+	throw_range = 7
+	w_class = WEIGHT_CLASS_NORMAL
+	force_unwielded = 8
+	force_wielded = 12
```

Comment:
12 force is the same as a toolbox.
Theoretically, if you and a tider with his blue toolbox were to robust eachother with perfect timing, you would both enter crit at the same time.
Is that intentional?

Reply:
two handed toolbox, the new meta

Reply:
It's also 12 force thrown. Ghetto ninja star, minus the embedding.

Reply:
this is a weird thing to get focused on but it was meant to be the same as the mop unwielded so i'll lower the throwforce

Reply:
Force just seems a bit high across the board. Shouldn't this have the same values as the mop?

Reply:
it does have the same force as the mop, it's slightly stronger when wielded with two hands

it's not a hill I'm going to die on but it's still literally the weakest two handed weapon right now

Reply:
mops are seriously strong, they should be the new meta.

Also toolboxes got nerfed down to this level, rather than these things getting buffed.